src/schema/contentPreference.ts

```diff
@@ -0,0 +1,285 @@
+import { IResolvers } from '@graphql-tools/utils';
+import { traceResolvers } from './trace';
+import { AuthContext, BaseContext } from '../Context';
+import { ContentPreference } from '../entity/contentPreference/ContentPreference';
+import { toGQLEnum } from '../common';
+import {
+  ContentPreferenceStatus,
+  ContentPreferenceType,
+} from '../entity/contentPreference/types';
+import { followEntity, unfollowEntity } from '../common/contentPreference';
+import { GQLEmptyResponse, offsetPageGenerator } from './common';
+import graphorm from '../graphorm';
+import { Connection, ConnectionArguments } from 'graphql-relay';
+
+export type GQLContentPreference = Pick<
+  ContentPreference,
+  'referenceId' | 'userId' | 'type' | 'createdAt' | 'status'
+>;
+
+export const typeDefs = /* GraphQL */ `
+  ${toGQLEnum(ContentPreferenceType, 'ContentPreferenceType')}
+
+  ${toGQLEnum(ContentPreferenceStatus, 'ContentPreferenceStatus')}
+
+  enum FollowStatus {
+    follow
+    subscribed
+  }
+
+  type ContentPreference {
+    referenceId: ID!
+
+    user: User!
+
+    type: ContentPreferenceType!
+
+    createdAt: DateTime!
+
+    status: ContentPreferenceStatus!
+  }
```

Comment:
Not sure if it was intentional or you missed it, non blocking, but this doesn't have descriptions

Reply:
We usually don't add descriptions to types, I think self explanatory?